controllers/managedfusionoffering_controller.go

```diff
@@ -46,9 +45,9 @@ type ManagedFusionOfferingReconciler struct {
 	Log    logr.Logger
 	Scheme *runtime.Scheme
 
-	ctx                   context.Context
-	namespace             string
-	managedFusionOffering *v1alpha1.ManagedFusionOffering
+	Ctx                   context.Context
+	Namespace             string
+	ManagedFusionOffering *v1alpha1.ManagedFusionOffering
```

Comment:
No reason to make this public. just pass it as a private param to the plugin reconcile code.
We assume the Embbeding-reconcilers have no direct knowledge of the specific of the embedded reconciler besides some interface that will include commons such as Ctx and maybe Namespace.